From eed2bd0aec1f7824f133da51c9987b346352e6d7 Mon Sep 17 00:00:00 2001
From: TimoDo <timo.capa@gmail.com>
Date: Sat, 15 Apr 2023 06:22:25 +0200
Subject: [PATCH 21/21] Launcher3: Enable FORCED_MONO_ICON by default

Enables Android to automatically generate monochromatic icons to use for monet.

Some icons can look weird but it's still better than a mismatched homescreen

Only turns on when Themed Icons is enabled, so this seems fine

Change-Id: I4a4b84051a4e09fda40faa438706c25958713f4a
Signed-off-by: Adithya R <gh0strider.2k18.reborn@gmail.com>
---
 src/com/android/launcher3/config/FeatureFlags.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/launcher3/config/FeatureFlags.java b/src/com/android/launcher3/config/FeatureFlags.java
index 96630db59f..dfaf642ec4 100644
--- a/src/com/android/launcher3/config/FeatureFlags.java
+++ b/src/com/android/launcher3/config/FeatureFlags.java
@@ -152,7 +152,7 @@ public final class FeatureFlags {
 
     // TODO(Block 7): Clean up flags
     public static final BooleanFlag ENABLE_FORCED_MONO_ICON = getDebugFlag(270396209,
-            "ENABLE_FORCED_MONO_ICON", DISABLED,
+            "ENABLE_FORCED_MONO_ICON", ENABLED,
             "Enable the ability to generate monochromatic icons, if it is not provided by the app");
 
     // TODO(Block 8): Clean up flags
-- 
2.34.1

